testplan/testing/multitest/base.py

```diff
@@ -937,6 +950,13 @@ def _run_serial_testcases(self, testsuite, testcases):
                         testsuite.name,
                     )
                     break
+            elif testcase_report.status == Status.FAILED:
```

Comment:
Because we are checking self.active in the for loop, so we can achieve by set self._should_abort instead of explicitly break? Also it might be better to set self._should_abort in _run_testcase method, rather than need to do it in both _run_serial_testcases and _run_parallel_testcases